crypto/fipsmodule/ec/ec_key.c

```diff
@@ -325,6 +326,42 @@ int EC_KEY_check_key(const EC_KEY *eckey) {
   return 1;
 }
 
+static int EVP_EC_KEY_check_fips(EC_KEY *key) {
+  uint8_t msg[16] = {0};
+  size_t msg_len = 16;
+  int ret = 0;
+  uint8_t* sig_der = NULL;
+  EVP_PKEY* evp_pkey = EVP_PKEY_new();
+  EVP_MD_CTX* ctx = EVP_MD_CTX_new();
+  const EVP_MD *hash = EVP_sha256();
+  size_t sign_len;
+  if (!evp_pkey ||
+      !ctx ||
+      !EVP_PKEY_set1_EC_KEY(evp_pkey, key) ||
+      !EVP_DigestSignInit(ctx, NULL, hash, NULL, evp_pkey) ||
+      !EVP_DigestSign(ctx, NULL, &sign_len, msg, msg_len)) {
+    goto err;
+  }
+  sig_der = OPENSSL_malloc(sign_len);
+  if (!sig_der ||
+      !EVP_DigestSign(ctx, sig_der, &sign_len, msg, msg_len)) {
+    goto err;
+  }
+  #if defined(BORINGSSL_FIPS_BREAK_ECDSA_PWCT)
+    msg[0] = ~msg[0];
+  #endif
```

Comment:
Is this needed in this test? I cannot see that it was there from before.

Reply:
Yes. Below link was before.
https://github.com/awslabs/aws-lc/blame/736fce4b11417ea07ae0df06913d7e87a3cd58ca/crypto/fipsmodule/ec/ec_key.c#L342-L344

Reply:
I see it now, thanks.